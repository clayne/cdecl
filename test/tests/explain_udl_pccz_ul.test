cdecl @ /dev/null @ -xc++ @ explain int operator"" _x(const char*, unsigned long) @ 0
