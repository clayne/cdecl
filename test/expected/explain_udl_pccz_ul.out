declare _x as user-defined literal (pointer to constant char, unsigned long int) returning int
